### DISCUSSION TITLE
How to do 'OUTPAINTING' in FORGE

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/stable-diffusion-webui-forge#1339] [General]
Simplifying the topic - 
How to perform outpainting as it is done in Fooocus?
-
Is there any specific extensions to perform it within the forge?

Thank You

**Comment by sclous** [comment]
Would love to know the same, only reason still using Fooocus is the outpainting which is amazing.

**Comment by mold0101** [comment]
![inpaint](https://github.com/user-attachments/assets/358cdaec-3bb1-484f-aac5-d809b94d4f94)

https://huggingface.co/lllyasviel/fooocus_inpaint/tree/main
